features/step_definitions/submit_review.rb

```diff
@@ -3,7 +3,7 @@
 Given(/^I'm on the new review page$/) { visit new_review_path }
 
 And(/^I enter "([^"]*)" in the "Overall rating" field$/) do |_value|
-  page.choose(:review_overall_rating_1)
+  page.choose('review_overall_rating_' + _value)
```

Comment:
Remove the underscore from `_value` in this, we are using the variable.

Reply:
Fixed